.github/workflows/ci.yml

```diff
@@ -0,0 +1,58 @@
+name: CI
+on:
+  pull_request:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+jobs:
+  CI:
+    runs-on: ubuntu-latest
+    permissions:
+      id-token: write
+      contents: read
+    steps:
+      # Checkout the branch
+      - uses: actions/checkout@v3
+      # Java is needed for the Scala Play app
+      - uses: actions/setup-java@v3
+        with:
+          java-version: '11'
+          distribution: 'corretto'
+          cache: 'sbt'
+      # Build CDK and Play (in sequence)
+      - run: ./scripts/ci.sh
+        shell: bash
+      # Fetch AWS credentials, allowing us to upload to Riff-Raff (well, S3)
+      - uses: aws-actions/configure-aws-credentials@v2
+        with:
+          role-to-assume: ${{ secrets.GU_RIFF_RAFF_ROLE_ARN }}
+          aws-region: eu-west-1
+      # Upload our build artifacts to Riff-Raff (well, S3)
+      - uses: guardian/actions-riff-raff@v2
+        with:
+          app: geoip-db-refresher
```

Comment:
The name `ophan::geoip-db-refresher` is new, but there already exists a `geoip-db-refresher`. I was happy to change it so it follows convention, but keep the build numbers. What would you recommend?

Reply:
@akash1810, I've made this change. Although I have now muddied the water with builds for both, can we remove the `ophan::geoip-db-refresher` entries. Are you otherwise happy with the PR?

Reply:
> can we remove the `ophan::geoip-db-refresher` entries.

Sure, I've deleted the files from S3 now.